OSDOCS-10895:RN for ANP (#77464)
openshift/openshift-docs: release_notes/ocp-4-16-release-notes.adoc

For more information, see xref:../networking/ovn_kubernetes_network_provider/converting-to-dual-stack.html#nw-dual-stack-convert_converting-to-dual-stack[Converting to a dual-stack cluster network] in _Converting to IPv4/IPv6 dual-stack networking_.

[id="ocp-4-16-anp"]
=== AdminNetworkPolicy is generally available

> This feature provides two new APIs, `AdminNetworkPolicy` (ANP) and `BaselineAdminNetworkPolicy` (BANP). Cluster Administrators can use ANP and BANP to apply cluster-scoped policies and safeguards for an entire cluster before namespaces are created. Because it is cluster scoped, ANP provides Administrators a solution to manage the security of their network at scale through multi-tenant isolation.

Reviewer: ```suggestion
This feature provides two new APIs, `AdminNetworkPolicy` (ANP) and `BaselineAdminNetworkPolicy` (BANP). Cluster Administrators can use ANP and BANP to apply cluster-scoped network policies and safeguards for an entire cluster before namespaces are created. Because it is cluster scoped, ANP provides Administrators a solution to manage the security of their network at scale without having to duplicate their network policies on each namespace.
```

Author: This is good. Thank you.

> This feature provides two new APIs, `AdminNetworkPolicy` (ANP) and `BaselineAdminNetworkPolicy` (BANP). Cluster Administrators can use ANP and BANP to apply cluster-scoped network policies and safeguards for an entire cluster before namespaces are created. Because it is cluster scoped, ANP provides Administrators a solution to manage the security of their network at scale without having to duplicate their network policies on each namespace.

Reviewer 2: Is it OK to refer to (B)ANP as API instead of Custom Resource or Objects? @tssurya 
It is part of policy API.

> This feature provides two new APIs, `AdminNetworkPolicy` (ANP) and `BaselineAdminNetworkPolicy` (BANP). Cluster Administrators can use ANP and BANP to apply cluster-scoped network policies and safeguards for an entire cluster before namespaces are created. Because it is cluster scoped, ANP provides Administrators a solution to manage the security of their network at scale without having to duplicate their network policies on each namespace.

Reviewer 3: Couple things, suggested rephrase:

This feature provides two new APIs, `AdminNetworkPolicy` (ANP) and `BaselineAdminNetworkPolicy` (BANP). Before namespaces are created, cluster administrators can use ANP and BANP to apply cluster-scoped network policies and safeguards for an entire cluster. Because it is cluster-scoped, ANP provides administrators with a solution to manage their network's security at scale without duplicating their network policies on each namespace.
